src/ocamlorg_web/lib/handler.ml

```diff
@@ -490,7 +478,7 @@ let package_doc t kind req =
                  | `ClassType s -> s)
             |> String.concat "."
           in
-          let title =
+          let _title =
```

Comment:
`_title` is never used.

Two possible ways to deal with that:
1. move the logic that decides the title of the page depending on is_latest_version over here
2. remove `_title` and keep the title rendering logic in the template